### ISSUE TITLE
notification.acceptPrimaryAction should be in the command palette

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#177873]
Testing #177672

and have a full name. Also, I think it doesn't even need the accessibility mode "when" clause, it seems generally useful to me.
